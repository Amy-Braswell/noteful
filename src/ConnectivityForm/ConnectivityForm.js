import React from 'react'

export default function ConnectivityfulForm(props) {
    const {className, ...otherProps} = props
    return(
        <form
            className={['Connectivity-form', className].join(' ')}
            action='#'
            {...otherProps}
        />
    )
}

