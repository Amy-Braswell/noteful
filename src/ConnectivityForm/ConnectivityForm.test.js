import React from 'react'
import ConnectivityForm from './ConnectivityForm'
import renderer from 'react-test-renderer'
import ReactDOM from 'react-dom'


it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<ConnectivityForm />, div)
    ReactDOM.unmountComponentAtNode(div)
})

describe(`ConnectivityForm component`, () => {
  const props = {
    className: 'test-class-name',
    children: <p>test children</p>,
    'data-other': 'test-other-prop'
  }

  it('renders a form.ConnectivityForm by default', () => {
    const tree = renderer 
    .create(<ConnectivityForm />)
    .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('renders the ConnectivityForm given props', () => {
    const tree = renderer
    .create(<ConnectivityForm {...props} />)
    .toJSON()
    expect(tree).toMatchSnapshot()
  })
})