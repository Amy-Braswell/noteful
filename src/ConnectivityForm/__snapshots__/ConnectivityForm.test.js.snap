// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConnectivityForm component renders a form.ConnectivityForm by default 1`] = `
<form
  action="#"
  className="Connectivity-form "
/>
`;

exports[`ConnectivityForm component renders the ConnectivityForm given props 1`] = `
<form
  action="#"
  className="Connectivity-form test-class-name"
  data-other="test-other-prop"
>
  <p>
    test children
  </p>
</form>
`;
